ros_connect.bash

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+echo "Be sure to call this script: source ./ros_connect {ROS_MASTER_IP} or source ./ros_connect {ROS_MASTER_IP} {ROS_TALKER_IP}"
+echo $#
+if [ $# == 2 ]
+then 
+    #Both ips provided
+    ROS_MASTER_URI="http://$1:11311"
+    ROS_IP=$2 
+    export ROS_MASTER_URI
+    export ROS_IP
+elif [ $# == 1 ] 
+then 
+    my_ip=`hostname -I |  awk '{print $1;}'` #Assumes the first ip returned is the one needed
+    echo "Your IP: " $my_ip
+    echo "ROS MASTER IP: " $1
+    ROS_MASTER_URI="http://$1:11311"
+    ROS_IP=$my_ip 
+    export ROS_MASTER_URI
+    export ROS_IP
+    echo "Done setting ROS_MASTER_URI and ROS_IP"
+
+else
+    echo "No IP address of master ROS computer provided"
```

Comment:
Let me tomorrow check if we can use the hotspot ip (#.#.#.1) by default when the hotspot is the master.

Reply:
That would be a good idea! I am reading the wiki for more info

Reply:
Yes this works. Then, please try to add in the default case to set the `ROS_MASTER_URI` with the first assignable address of the lan, i.e. if network address is `127.10.0.0`, then will be `127.10.0.1`. This could be done with a little logic using network address and mask.